libraries/Microsoft.Bot.Schema/Teams/TargetedMeetingNotification.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Specifies Teams targeted meeting notification.
+    /// </summary>
+    public class TargetedMeetingNotification : BotMeetingNotification<TargetedMeetingNotificationValue>
```

Comment:
Would it be useful to provide a constructor for this class that takes in an TargetedMeetingNotificationValue as a parameter?

We can also maintain an empty constructor

Reply:
oh @vule-msft , this is just for my own learning :) 
What are the potential benefits of providing a constructor here over the current design? 
Ex: More explicit intention to developers that they need to set TargetedMeetingNotifciationValue?

Reply:
Just to provide developers an easy way to create an instance and set the value. I anticipate, in most cases, developers will want to set `value` upon instance creation.

```C#
var targetedMeetingNotif = new TargetedMeetingNotification(targetedMeetingNotificationValue);
// vs.
var targetedMeetingNotif = new TargetedMeetingNotification() {Value = targetedMeetingNotificationValue};
```

Let's check with the SDK team to see what they think

Reply:
Updated in the next iteration. Please correct me if I understand it wrong. 
Besides, will close other comments and leave this comment open here so that Bot Framework can have a review.

Reply:
Oh i see your point, @vule-msft. Vu, could you verify whether my understanding aligns with yours?
 
So with the existing approach (no explicit constructor, which is the same as empty constructor), this is what a developer needs to do: 
`var targetedMeetingNotif = new TargetedMeetingNotification() {Value = targetedMeetingNotificationValue};`

But with your proposal (a constructor for this class that takes in an TargetedMeetingNotificationValue as a parameter), a developer can do this instead:
`var targetedMeetingNotif = new TargetedMeetingNotification(targetedMeetingNotificationValue);`

Assuming my understanding is correct... I see the benefits on developer side with your proposed approach. 

Another idea after I properly understood your comment is: maybe we can provide both constructors(one empty, and one with parameter) to provide flexibility to developers?  But I do not have strong opinion on this. We can choose the approach that the existing BF SDK provides to developers :)

Reply:
```
Another idea after I properly understood your comment is: maybe we can provide both constructors(one empty, and one with parameter) to provide flexibility to developers? But I do not have strong opinion on this
```

Yep, this is what I meant. We can have both constructors

Reply:
@yingduyingdu - are we planning to add?

Reply:
@vule-msft updated in the next iteration. Could you please help review the latest change and confirm if it is what you expected? Thanks!

Reply:
Vu confirmed the change offline.